crates/bevy_render/src/extract_component.rs

```diff
@@ -209,12 +210,12 @@ impl<T: Asset> ExtractComponent for Handle<T> {
 fn extract_components<C: ExtractComponent>(
```

Comment:
I think need to give more attention to ExtractComponent since it opens us up to leaving unsynchronized components lying around. Actually this applies to all of our other systems where filters are involved, though I haven't had the chance to go through all of those yet. For example, if an entity matches the ExtractComponent query filter in one frame and the next it doesn't, we're potentially leaving garbage data around.

Reply:
yeah Currently, it appears that ExtractComponentPlugin is only used for extracting components used in Camera View. But it can indeed be used to extract components from other entities as well. Perhaps we should give it a different name to better indicate its usage?